README.md

```diff
@@ -11,4 +11,7 @@ https://www.microsoft.com/en-us/download/details.aspx?id=6812
 It would be nice to modernize the DirectX usage.  Some notes about replacement packages are available here:
 https://stackoverflow.com/questions/33873592/where-i-can-find-microsoft-directx-assembly-to-reference
 
-Sample configuration files are included in the config directory.  To see all the puzzles, you will want to copy these to the directory where the built executable lives.
```

Comment:
Since we can automatically copy the config files to output directory, we don't need to mention it in README.